packages/pangea-node-sdk/src/services/authn/agreements/index.ts

```diff
@@ -6,34 +6,33 @@ import { AuthN } from "@src/types.js";
 export default class Agreements extends BaseService {
   constructor(token: string, config: PangeaConfig) {
     super("authn", token, config);
-    this.apiVersion = "v1";
   }
 
   // TODO: Docs
   create(
     request: AuthN.Agreements.CreateRequest
   ): Promise<PangeaResponse<AuthN.Agreements.CreateResult>> {
-    return this.post("agreements/create", request);
+    return this.post("v2/agreements/create", request);
   }
 
   // TODO: Docs
   delete(
     request: AuthN.Agreements.DeleteRequest
   ): Promise<PangeaResponse<AuthN.Agreements.DeleteResult>> {
-    return this.post("agreements/delete", request);
+    return this.post("v2/agreements/delete", request);
   }
 
   // TODO: Docs
   update(
     request: AuthN.Agreements.UpdateRequest
   ): Promise<PangeaResponse<AuthN.Agreements.UpdateRequest>> {
-    return this.post("agreements/update", request);
+    return this.post("v2/agreements/update", request);
   }
 
   // TODO: Docs
   list(
     request: AuthN.Agreements.ListRequest
```

Comment:
@pangea-andrest how would we feel about making request an optional param that defaults to an empty object here?

Something like:
```suggestion
    request?: AuthN.Agreements.ListRequest = {}
```
(check if the above actually works though...)

I think this makes the developer experience a little nicer for the use case of not passing any optional params because this method could then be called like:
```
const response = await authn.agreements.list();
```
instead of _needing_ to be called like:
```
const response = await authn.agreements.list({});
```

Reply:
Sure! Thanks!